starport/cmd/build.go

```diff
@@ -19,14 +19,14 @@ const (
 func NewBuild() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "build",
-		Short: "Build your app",
-		Long: `By default, build command will build your application's binaries
-and add them to your $(go env GOPATH)/bin path.
+		Short: "Build a node binary",
+		Long: `By default, build your node binaries
+and add the binaries to your $(go env GOPATH)/bin path.
 
-If you want to only build binaries for a release, use the --release flag with
---release.targets (optional). Then, binaries built for chosen targets will appear
-in your app's path under release/. You can add any number of targets in GOOS:GOARCH format.
-If you don't provide any, a binary will be created for your own machine.
+To build binaries for a release, use the optional --release flag. The app binaries
+for one or more specified release targets are built in a release/ dir under the app's
+source. Specify the release targets with GOOS:GOARCH build tags.
+If the GOOS:GOARCH build tags are not specified, a binary is created for your local machine.
```

Comment:
```suggestion
If no --release.targets specified, a binary is created for the machine you run this command inside.
```

something like this might be leaner. I also wanted to avoid duplicating GOOS:GOARCH expression twice.

Reply:
wdyt about "your current environment"?